lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -2,22 +2,21 @@ const ChangeTypes = require('../../ChangeTypes');
 const Node = require('../../node');
 const TreeSearch = require('../../TreeSearch');
 
-// If `node` is a product of terms where some are fractions (but none are
-// polynomial terms), multiplies them together.
+// If `node` is a product of terms where:
+// 1) at least one is a fraction
+// 2) either none are polynomial terms, OR
+//    at least one has a symbol in the denominator
+// then multiply them together.
```

Comment:
Hm yeah I think that's it. I feel like that case could be better handled - cause right now 5/2 * x * y/5 won't be combined here... but I guess that'd be handled in collect and combine? Just wondering if we're leaving out anything else with this clause

But yeah adding that as a comment would be helpful for people seeing this later and wondering why, like we both just did haha